opentuner/search/technique.py

```diff
@@ -204,15 +223,20 @@ def call_main_generator(self):
                                     self.pending_tests)
         if self.pending_tests:
           self.rounds_since_novel_request = 0
-          yield None # wait
+          yield False # wait
 
 #list of all techniques
 the_registry = list()
 
+#list of composable techniques
+the_composable_registry = list()
 
 def register(t):
   the_registry.append(t)
 
+def register_composable(t):
```

Comment:
I wanted to deal with multiple registered versions of a ComposableEvolutionaryTechnique skewing probabilities of which one you end up picking to generate from. Is there a good way to do this?